CHANGELOG.md

```diff
@@ -361,6 +363,7 @@ The format is based on [Keep a Changelog].
 [#414]: https://github.com/raxod502/selectrum/pull/414
 [#420]: https://github.com/raxod502/selectrum/issues/420
 [#421]: https://github.com/raxod502/selectrum/pull/421
+[#438]: https://github.com/raxod502/selectrum/issues/438
```

Comment:
Sorry, one last thing I missed, also mention the PR #439 here and in the comment above

Reply:
Roger. 

Just out of curiosity are we expected to have seperate commits for updating the changelog and making actual changes? We make changes, open a PR, then update the changelog with a link to the issue and the PR.

Reply:
Yes, that would be the usual process, I squash the PRs anyway but adding the changelog after review is usually also better  because what needs to get added might change after that.

Reply:
All right. I'll take that approach in future. Thank you :smile:

Reply:
You are welcome :)